.github/workflows/bot-handler.yml

```
@@ -0,0 +1,177 @@
+name: Comprehensive Bot Handler
+
+on:
+  issues:
+    types: [opened, edited, reopened, labeled, unlabeled]
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review, labeled, unlabeled]
+  check_suite:
+    types: [completed]
+  status:
+  schedule:
+    - cron: '*/1 * * * *'  # Every minute
```

actionlint (1.7.7)

scheduled job runs too frequently. it runs once per 60 seconds. the shortest interval is once every 5 minutes

(events)